src/conf_mode/protocols_bgp.py

```diff
@@ -115,9 +117,11 @@ def verify(bgp):
                     if not verify_remote_as(peer_config, asn_config):
                         raise ConfigError(f'Neighbor "{peer}" remote-as must be set!')
 
+                    # Only checks for ipv4 and ipv6 neighbors
+                    if is_ipv4(peer) or is_ipv6(peer):
```

Comment:
Why not use the `is_ip()` helper?

It's implemented as:
```
def is_ip(addr):
    """ Check addr if it is an IPv4 or IPv6 address """
    return is_ipv4(addr) or is_ipv6(addr)
```

Reply:
Good point. Thanks.
Done.